charts/tempo/Chart.yaml

```
@@ -2,8 +2,8 @@ apiVersion: v2
 name: tempo
 description: Grafana Tempo Single Binary Mode
 type: application
-version: 1.18.1
-appVersion: 2.7.0
+version: 1.19.0
```

```suggestion
version: 1.18.2
```
@KyriosGN0 Thanks for the contribution! however I believe patch version would increase ?